src/nile/utils/__init__.py

```diff
@@ -1,157 +1,24 @@
 """Utilities for Nile scripting."""
-import math
-import re
-from pathlib import Path
-
-try:
-    from starkware.starknet.business_logic.execution.objects import Event
-    from starkware.starknet.public.abi import get_selector_from_name
-    from starkware.starkware_utils.error_handling import StarkException
-except BaseException:
-    pass
+# flake8: noqa
+
+from .miscellaneous import (
```

Comment:
I wanted to separate generic utils from test utils. Miscellaneous is certainly not required for this, I could move utils there to the `__init__.py` file, but I thought that a separate file would allow `__init__.py` to be more readable, for us to be aware of what functions are we publishing as generic utils.